permissions/component-pipeline-model-json-shaded.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "pipeline-model-json-shaded"
+paths:
+- "org/jenkinsci/plugins/pipeline-model-json-shaded"
```

Comment:
This groupId makes no sense for something that's not a plugin, just to save a line in the pom.